src/navs/index.js

```diff
@@ -4530,20 +4530,51 @@ export const docsMenu = {
                     name: 'Getting started',
                 },
                 {
-                    name: 'Getting started',
-                    url: '/docs/revenue-analytics/getting-started',
-                    icon: 'IconBook',
+                    name: 'Start here',
+                    url: '/docs/revenue-analytics/start-here',
+                    icon: 'IconListCheck',
+                    color: 'orange',
+                    featured: true,
+                },
+                {
+                    name: 'Payment platforms',
+                    url: '/docs/revenue-analytics/payment-platforms',
+                    icon: 'IconCode',
                     color: 'blue',
                     badge: {
                         title: 'Beta',
                         className: 'uppercase !bg-blue/10 !text-blue !dark:text-white !dark:bg-blue/50',
                     },
+                    children: [
```

Comment:
Should have an overview page and link to `/docs/revenue-analytics/payment-platforms`